charts/veritable-ui/README.md

```diff
@@ -0,0 +1,309 @@
+# veritable-ui
+
+The veritable-ui is a component of the Sequence (veritable) ledger-based system. The veritable-ui responsible for matching orders, and transactions as well as indexing blocks locally, it exposes API for this purpose. See [https://github.com/digicatapult/veritable-documentation](https://github.com/digicatapult/veritable-documentation) for details.
```

Comment:
Change the description

Reply:
Updated and should be reflected in the next push